### ISSUE TITLE
[Bug]: step certificate install --java fails with exit status 1

### ISSUE THREAD

**Issue by cowwoc** [smallstep/certificates#2024]
### Steps to Reproduce

Run `step certificate install root_ca.crt --java --no-system`

### Your Environment

* OS - Microsoft Windows [Version 10.0.19045.4894]
* `step-ca` Version -
```
Smallstep CLI/0.27.4 (windows/amd64)
Release Date: 2024-09-13T17:37:59Z
```

### Expected Behavior

Truststore to be updated

### Actual Behavior

failed to execute "C:\Program Files\Java\zulu-21\bin\keytool.exe -importcert -noprompt -keystore C:\Program Files\Java\zulu-21\lib\security\cacerts -storepass changeit -file C:\Users\Gili\AppData\Local\Temp\truststore.1319194278.pem -alias step-ca Root CA 105762468571059453893283164350196192992" failed with: exit status 1

### Contributing

Vote on this issue by adding a 👍 reaction.
To contribute a fix for this issue, leave a comment (and link to your pull request, if you've opened one already).

**Comment by cowwoc** [comment]
I suspect that the failure is caused by the fact that paths with spaces are not being quoted, but I don't know of a way to check if this is the case. The created pem file is deleted on failure so I cannot run the command-line directly.

**Comment by hslatman** [comment]
It seems you're using "Azul Zulu" Java? Do `C:\Program Files\Java\zulu-21\bin\keytool.exe` and `C:\Program Files\Java\zulu-21\lib\security\cacerts` actually exist, and are they in those locations?

The PEM that is created is a temporary file with the root CA certificate contents in it. You can specify that instead to test the command.

**Comment by cowwoc** [comment]
1. I converted `root_ca.crt` to `root_ca.pem` using `openssl x509 -in "%HOMEPATH%\.step\certs\root_ca.crt" -out "%HOMEPATH%\.step\certs\root_ca.pem" -outform PEM`
2. I replaced the temporary file with `root_ca.pem` as you suggested.
3. The command failed due to spaces in the paths (i.e. `Program Files`)
4. I had to quote the path to the executable, the path to cacerts, and the value of the alias. Once that was done, the command worked.

So to recap: it seems that you have a bug on Windows. The tool needs to surround those 3 elements in quotes.
